packages/tx/src/types.ts

```diff
@@ -287,4 +328,72 @@ export interface JsonRpcTx {
   v: string // QUANTITY - ECDSA recovery id
   r: string // DATA, 32 Bytes - ECDSA signature r
   s: string // DATA, 32 Bytes - ECDSA signature s
+  maxFeePerDataGas?: string // QUANTITY - max data fee for blob transactions
+  versionedHashes?: string[] // DATA - array of 32 byte versioned hashes for blob transactions
 }
+
+/** EIP4844 constants */
+
+// TODO: Decide which of these should be in the hardfork params instead of hardcoded as constants here
+export const BLOB_COMMITMENT_VERSION_KZG = 0x01
+export const MAX_CALLDATA_SIZE = 16777216 // 2 ** 24
+export const MAX_ACCESS_LIST_SIZE = 16777216 // 2 ** 24
+export const MAX_VERSIONED_HASHES_LIST_SIZE = 16777216 // 2 ** 24
+export const LIMIT_BLOBS_PER_TX = 2
```

Comment:
Ah, good catch.  In practice, the limit is currently 4 since that's the equivalent of the maximum data gas that can be included in a single block so I i'm not sure why it's set so high in the spec.  I will update this.